vendor/knative.dev/eventing-kafka/test/e2e/helpers/kafka_helper.go

```diff
@@ -51,7 +51,7 @@ const (
 	strimziTopicResource = "kafkatopics"
 	interval             = 3 * time.Second
 	timeout              = 4 * time.Minute
-	kafkaCatImage        = "quay.io/openshift-knative/kafkacat:1.6.0"
+	kafkaCatImage        = "quay.io/openshift-knative-eventing-kafka-test/kafkacat:1.6.0"
```

Comment:
You're actually changing stuff in the vendor directory. This is not allowed and the `Validate / Generated files are commited` check fails on this PR. This change will need to be done here: https://github.com/openshift-knative/eventing-kafka/blob/release-v1.1/test/e2e/helpers/kafka_helper.go#L54 (branch release-v1.1).

Reply:
Reverted to original.